kickstarter/android-oss
# WEB-833: CTA Clicked event on creator dashboard "select another project" button

# 📲 What

Create segment CTA Clicked event when a creator taps "select another project" on the creator dashboard.

# 🤔 Why

The native audit initiative

# 👀 See

<img width="490" alt="Screen Shot 2022-12-21 at 3 15 39 PM" src="https://user-images.githubusercontent.com/19390326/208995175-542b2e7b-e003-42ea-b152-0414a1112378.png">

# 📋 QA

- Log into a creator account on staging
- Select "Dashboard" from the overflow menu
- Tap select another project
- Observe event fired in segment (make sure you are on the staging repository on segment)

# Story 📖

[WEB-833: CTA Clicked event on creator dashboard "select another project" button](https://kickstarter.atlassian.net/browse/WEB-833)
